packages/psss-server/src/connections/MeditrakConnection.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { ApiConnection } from './ApiConnection';
+
+const { MEDITRAK_API_URL = 'http://localhost:8090/v2' } = process.env;
+
+export class MeditrakConnection extends ApiConnection {
+  baseUrl = MEDITRAK_API_URL;
+}
```

Comment:
Although this pattern looks cool, I'm not sure if we need this and `ReportConnection` as separate classes: we could just pass `baseUrl` in the constructor.

Unless if we want to add custom behaviour in these classes in the very near future

Reply:
Yep good point, but indeed the plan is that each will be filled out with some more custom behaviour as we implement various psss endpoints that need to connect. If they're still empty in two weeks, I agree we should change the pattern!